### DISCUSSION TITLE
Are upcoming stable releases announced in advance?

### DISCUSSION THREAD

**Discussion by Gasparmejor** [AzuraCast/AzuraCast#7478] [Q&A] [answered]
Hi all,

I am currently on the Rolling Release channel and my Azuracast installation is automatically updated weekly through a cronjob. Because the station i'm building as a hobby is getting closer to being possibly ready for the outside world, I would like to switch to the Stable Release channel. I'm anxious to miss my window of opportunity when a stable version would be released, but quickly followed by a new rolling release. Hence the question: are upcoming stable releases announced in advance, and if so, where?

Thanks a lot en have a nice day!

**Comment by BusterNeece** [comment, marked as answer]
As of the last few stable releases, we've had a somewhat less-advertised `./docker.sh rollback (version)` command that you can use to migrate your installation _backward_ to the stable release you specify, reversing any database changes that have been made since that release's date.

This should largely mitigate any concerns you might have about being able to jump from Rolling to Stable. We also try to intentionally avoid making large database migrations in the immediate aftermath of a Stable release (though this is sometimes unavoidable).

Because our stable release timing depends on so many outside factors, it's very difficult for us to say authoritatively ahead of time when a new stable release will be out.

Currently, we are expecting to tag a stable release once the immediate work on our AutoDJ scheduler has completed and been established as stable for a little while, and before the 2.3.x release of Liquidsoap, which we'll use our next Rolling cycle to test fully.

**Reply by Gasparmejor** [reply]
Thanks for your reply Buster. Have a great day!
